acm/registration/near-edge/base/near-edge.yaml

```diff
@@ -1,39 +1,36 @@
 ---
-apiVersion: app.k8s.io/v1beta1
-kind: Application
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
 metadata:
-  name: application
-  namespace: near-edge-acm-template
+  name: namespace-policy
+  namespace: openshift-gitops
```

Comment:
This per-application `Policy` resource doesn't really need to be in `openshift-gitops`, and maybe this isn't the best place for it on the hub. Open to suggestions around that, whether it's okay, or whether it should go to some other namespace. I mostly chose here for now, as that's where the closely-related ApplicationSet resource needs to be.